gap/examples.gi

```diff
@@ -370,3 +370,38 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(BananaTreeGraphCons,
+"for IsMutableDigraph and two positive integers",
+[IsMutableDigraph, IsPosInt, IsPosInt],
+function(filt, m, n)
+  local D, i, j;
+  D := EmptyDigraph(IsMutable, 1);
+  for i in [1 .. m] do
+    D := DigraphDisjointUnion(D, CompleteBipartiteDigraph(IsMutable, 1, n - 1));
+  od;
+  for j in [0 .. (m - 1)] do
+    DigraphAddEdges(D, [[1, (j * n + 3)], [(j * n + 3), 1]]);
+  od;
+  return D;
+end);
+
+InstallMethod(BananaTreeGraphCons,
+"for IsImmutableDigraph and two positive integers",
+[IsImmutableDigraph, IsPosInt, IsPosInt],
+function(filt, m, n)
+  local D;
+  D := MakeImmutable(BananaTreeGraphCons(IsMutableDigraph, m, n));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
```

Comment:
You could set some other properties here (the obvious one is `IsAcyclicDigraph`)

Reply:
@flsmith, non-empty symmetric digraphs are not acyclic!

Perhaps you're thinking of `IsUndirectedTree`.